exchanges/kraken/kraken_websocket.go

```diff
@@ -941,8 +943,10 @@ func (k *Kraken) wsProcessOrderBookPartial(channelData *WebsocketChannelData, as
 		}
 
 		base.Bids[i] = orderbook.Item{
-			Amount: amount,
-			Price:  price,
+			Amount:    amount,
+			StrAmount: bids[1].(string),
+			Price:     price,
+			StrPrice:  bids[0].(string),
```

Comment:
An acceptable spot for nolint given that you wouldn't get here due to the above `ParseFloats`